pretix_closer2event/signals.py

```diff
@@ -107,6 +107,9 @@ def signal_process_response(
         h = _parse_csp(response["Content-Security-Policy"])
     else:
         h = {}
+    if getattr(request, 'pci_dss_payment_page', False):
+        # No tracking scripts on PCI DSS relevant payment pages
+        return response
```

Comment:
As closer2event ist anyways only embedded on order_info, this could be optimized so that CSP-header is only sent on order_info-pages. But probably not that easy to implement, so leave it that way. Maybe as micro optimization move it to the top of this function?